packages/tracing-internal/src/browser/browsertracing.ts

```diff
@@ -278,7 +286,7 @@ export class BrowserTracing implements Integration {
       tracePropagationTargets,
       shouldCreateSpanForRequest,
       _experiments: {
-        enableHTTPTimings: _experiments.enableHTTPTimings,
```

Comment:
can we update the func signature of instrumentOutgoingRequests to take  `enableHTTPTimings`?

Some people do use `instrumentOutgoingRequests` outside of `BrowserTracing`.